docs/UserGuide.md

```diff
@@ -119,11 +119,13 @@ Format: `edit INDEX NEWTASK`<br>
  <br> <img src="images/editCmd.png" width="600"> </br>
 Picture 7: `edit 2 schedule meeting on wednesday: edits the 2nd task in Taskell to "schedule meeting on wednesday."`
 
-#### Saving the information in Taskell
+#### Saving the information in Taskell : `save`
 
-Specify the path of a new folder to store the data file of Taskell. The user should have permissions to access the folder.<br>
-Format: `save /FOLDERPATH`<br>
-Example: `save /Jim/Documents`
+Specify the path of a new folder to store the data file of Taskell. The user should have permissions to access the folder. <br>
+If directory is valid but file is missing, for example if command is `save C:/Users/Jim/Documents/chicken`,  and `C:/Users/Jim/Documents/chicken` is valid
+but `chicken` file is not created, Taskell will create the file.<br>
```

Comment:
Suggestion (Use less of passive words):
Specify the path <strong>to a new folder where you want</strong> ~~of a new folder~~ to store ~~the~~ <strong>your</strong> data file of Taskell. ~~The user~~ <strong>You</strong> should have permission~~s~~to access ~~the~~ <strong>this</strong> folder. <br>
 Format: `save /FOLDERPATH`<br>
If <strong>the specified</strong> directory is valid but <strong>the</strong> file is missing, for example if command is `save C:/Users/Jim/Documents/chicken`,  and `C:/Users/Jim/Documents/chicken` is valid but `chicken` file is not created, Taskell will create the file <strong>for you</strong>.<br>
 Example: `save /Jim/Documents`